common/djangoapps/third_party_auth/tests/test_settings.py

```diff
@@ -13,7 +13,11 @@
     'AUTHENTICATION_BACKENDS': _ORIGINAL_AUTHENTICATION_BACKENDS,
     'INSTALLED_APPS': _ORIGINAL_INSTALLED_APPS,
     'MIDDLEWARE_CLASSES': _ORIGINAL_MIDDLEWARE_CLASSES,
-    'TEMPLATE_CONTEXT_PROCESSORS': _ORIGINAL_TEMPLATE_CONTEXT_PROCESSORS,
+    'TEMPLATES': [{
+        'OPTIONS': {
+            'context_processors': _ORIGINAL_TEMPLATE_CONTEXT_PROCESSORS
+        }
+    }],
```

Comment:
Yes - the `apply_settings()` calls below add a few context processors. So this section of the fake settings needs to be present.